tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist=docs,py{38,39,310,311,312,313,py3},py{38}-pytest{main}
+envlist=docs,py{39,310,311,312,313,py3},py{39}-pytest{main}
```

Comment:
Do we remember why main Is in brackets?

Reply:
No, I also found that odd. Will remove it.

Reply:
Done.